components/centraldashboard/ci-centraldashboard-pipeline-run.yaml

```diff
@@ -0,0 +1,393 @@
+---
```

Comment:
Can we move the deployment files to `components/centraldashboard/ci-cd` to separate them from the application files? 
This can then form a standard for organizing other components as well.
If this is too much trouble, it can stay the way it is.